### ISSUE TITLE
dock in all my monitors at the same time ?

### ISSUE THREAD

**Issue by aniel300** [mydockfinder/mydockfinder#432]
how can i have the dock in all my monitors at the same time?

**Comment by mydockfinder** [comment]
不行，多屏同时显示几乎不可能，因为dock独立渲染很占GPU，但是统一渲染又不好计算图标位置




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年9月20日(星期二) 晚上8:27
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)






我怎么能有码头在我所有的显示器在同一时间？

—
直接回复此邮件，在GitHub上查看，或取消订阅.
***@***.***与&gt;.

**Comment by aniel300** [comment]
can u make the transition/ animation from one screen to the other smoother then?

**Comment by aniel300** [comment]
clicking exit on chrome from the dock make it that when i reopen it complains that it was force closed

**Comment by aniel300** [comment]
what about this ? 
![image](https://user-images.githubusercontent.com/7571729/191355003-212166e2-6465-4180-ac0c-93630fbe5196.png)

**Comment by mydockfinder** [comment]
鼠标按住了菜单别放开鼠标左键，然后鼠标离开菜单松开鼠标左键，然后点击其他地方就可以隐藏菜单了&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年9月21日(星期三) 凌晨4:10
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
what about this ?

**Comment by aniel300** [comment]
> what about this ? ![image](https://user-images.githubusercontent.com/7571729/191355003-212166e2-6465-4180-ac0c-93630fbe5196.png)

why are the menus hidden ?

**Comment by aniel300** [comment]
can u make it so that icons don't rearrange from time to time and also make a option to have dock on every monitor ?

**Comment by mydockfinder** [comment]
真的不好弄，我需要一些时间研究一下




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年9月21日(星期三) 中午1:53
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
can u make it so that icons don't rearrange from time to time and also make a option to have dock on every monitor ?
 
&mdash;

**Comment by aniel300** [comment]
> 

ok thanks

**Comment by aniel300** [comment]
sw is crashing sometimes 
![q0gPdl8BKk](https://user-images.githubusercontent.com/7571729/191533858-346542c5-60b1-443e-bb38-cddc650b0905.png)
![30MmxLJt3g](https://user-images.githubusercontent.com/7571729/191533876-24418962-3c67-4f56-be7c-5d38f8d61ba1.png)

**Comment by mydockfinder** [comment]
是在home菜单会卡住吗



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年9月21日(星期三) 晚上10:37
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
sw is crashing sometimes
 
 
 
&mdash;

**Comment by aniel300** [comment]
had to force close it

**Comment by aniel300** [comment]
i dont think this is right 
![image](https://user-images.githubusercontent.com/7571729/191630047-a30ed364-dba5-48c8-9238-ad7cf3b88b1b.png)

**Comment by mydockfinder** [comment]
是的。。。。。。




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年9月22日(星期四) 上午7:57
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
i dont think this is right
 
 
&mdash;

**Comment by aniel300** [comment]
another crash
![image](https://user-images.githubusercontent.com/7571729/191658451-d0d6c8de-7efe-4600-af76-be0a5cd4aeb7.png)

**Comment by mydockfinder** [comment]
。。。。。。。。




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年9月22日(星期四) 中午12:28
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
another crash
 
 
&mdash;

**Comment by aniel300** [comment]
are u working on a new update ?

**Comment by mydockfinder** [comment]
yes




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年9月23日(星期五) 凌晨5:12
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
are u working on a new update ?
 
&mdash;

**Comment by aniel300** [comment]
when i click on quit is force close my apps, can u change that ? and when is the new update coming ?

**Comment by FulytheFox** [comment]
Hey, I don't know if it's the same bug, but MyFinder freezes when a window is maximized on another screen. The freeze only occours when `Maximized windows affect Finder appearance` is enabled. Here is a small demo:

https://user-images.githubusercontent.com/37185496/193091957-0f6d3852-2e48-4dc5-98a1-29aeb584c4e4.mp4

I hope this helps you to fix the bug!

**Comment by mydockfinder** [comment]
只有主屏有沉浸模式，其他屏幕没有，因为太难了我不会。。。。。




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年9月30日(星期五) 凌晨1:04
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
Hey, I don't know if it's the same bug, but MyFinder freezes when a window is maximized on another screen. The freeze only occours when Maximized windows affect Finder appearance is enabled. Here is a small demo:
 
https://user-images.githubusercontent.com/37185496/193091957-0f6d3852-2e48-4dc5-98a1-29aeb584c4e4.mp4
 
I hope this helps you to fix the bug!
 
&mdash;

**Comment by mydockfinder** [comment]
不行，因为每个程序的退出方式都不一样，退出就是任务管理器里面的强制退出




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年9月23日(星期五) 晚上7:18
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
when i click on quit is force close my apps, can u change that ? and when is the new update coming ?
 
&mdash;

**Comment by aniel300** [comment]
u dont speak english at all ?

**Comment by mydockfinder** [comment]
no




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年10月13日(星期四) 上午9:37
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
u dont speak english at all ?
 
&mdash;

**Comment by aniel300** [comment]
:(

**Comment by mydockfinder** [comment]
但是有翻译，如果你翻译成英文看不懂，我可以自己翻译英文然后发给你，会好一点




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年10月13日(星期四) 下午3:56
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
:(
 
&mdash;

**Comment by aniel300** [comment]
yes i can use google translator no problem. also have u been able to add the option to have the dock on multiple displays at the same time ?

**Comment by mydockfinder** [comment]
No, because the dock is rendered independently, and the effect of icon enlargement, multiple screens rendering at the same time is a lot of burden on the GPU. And independent algorithms are also difficult, which is why Mac OS has not been on the same screen to display the dock at the same time




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年10月14日(星期五) 上午8:05
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock in all my monitors at the same time ? (Issue #432)





 
yes i can use google translator no problem. also have u been able to add the option to have the dock on multiple displays at the same time ?
 
&mdash;

**Comment by aniel300** [comment]
umm that is too bad
